lib/oneview-sdk/image_streamer.rb

```diff
@@ -0,0 +1,56 @@
+# (c) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative 'version'
+require_relative 'exceptions'
+require_relative 'image-streamer/client'
+require_relative 'resource'
+Dir[File.dirname(__FILE__) + '/image-streamer/resource/*.rb'].each { |file| require file }
+
+# Module for interacting with the HPE OneView API
+module OneviewSDK
+  # Module for interacting with the Image Streamer
+  module ImageStreamer
+    ENV_VARS = %w(ONEVIEWSDK_I3S_URL ONEVIEWSDK_I3S_TOKEN ONEVIEWSDK_I3S_SSL_ENABLED).freeze
```

Comment:
These are probably OK here, but I'd probably prefer them to all live together in [onview-sdk.rb](https://github.com/HewlettPackard/oneview-sdk-ruby/blob/master/lib/oneview-sdk.rb#L21).